README.md

```diff
@@ -1,8 +1,11 @@
 # Lanelet2
 
-| [ROS focal/noetic](https://build.ros.org/job/Ndev__lanelet2__ubuntu_focal_amd64/lastBuild) | [ROS focal/foxy](https://build.ros2.org/job/Fdev__lanelet2__ubuntu_focal_amd64/lastBuild) |
-| --------- | --------- |
-| [![](https://build.ros.org/job/Ndev__lanelet2__ubuntu_focal_amd64/lastBuild/badge/icon)](https://build.ros.org/job/Ndev__lanelet2__ubuntu_focal_amd64/lastBuild) | [![](https://build.ros2.org/job/Fdev__lanelet2__ubuntu_focal_amd64/lastBuild//badge/icon)](https://build.ros2.org/job/Fdev__lanelet2__ubuntu_focal_amd64/lastBuild) |
+[![Build Status ROS focal/noetic](https://build.ros.org/job/Ndev__lanelet2__ubuntu_focal_amd64/lastBuild/badge/icon?subject=ROS%20noetic%20%28on%20focal%29)](
+https://build.ros.org/job/Ndev__lanelet2__ubuntu_focal_amd64/lastBuild/)
+[![Build Status ROS2 focal/foxy](https://build.ros2.org/job/Fdev__lanelet2__ubuntu_focal_amd64/lastBuild/badge/icon?subject=ROS2%20foxy%20%28on%20focal%29)](
```

Comment:
Foxy is already EOL in May. We should probably also include humble on jammy.

Reply:
done in [df61246](https://github.com/fzi-forschungszentrum-informatik/Lanelet2/pull/284/commits/df61246f8eb05ad22fb9f8277942fe9052702cf0)